core/templates/services/playthrough-issues-backend-api.service.ts

```diff
@@ -89,7 +89,7 @@ export class PlaythroughIssuesBackendApiService {
     return this.httpClient.get(
       this.getFullPlaythroughUrl(explorationId, playthroughId), {
         observe: 'response'
-      }).toPromise().then((response: HttpResponse<PlaythroughIssue>) => {
+      }).toPromise().then((response: HttpResponse<any>) => {
```

Comment:
Please also add the TODO to change this to a specific type.

Reply:
For Httpresponse I have made these changes